Dockerfile

```diff
@@ -19,8 +19,11 @@ COPY bin/generate-registry.sh ${wd}/bin/
 # download jq
 RUN curl -L -o /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 \
   && chmod +x /usr/local/bin/jq
-# build cargo-local-registry
-RUN cargo install --git https://github.com/ChrisGreenaway/cargo-local-registry.git
+# retrieve cargo-local-registry
+RUN curl -L -o clr.tar.gz https://github.com/dhovart/cargo-local-registry/releases/download/0.2.2/cargo-local-registry-0.2.2-x86_64-unknown-linux-musl.tar.gz && \
```

Comment:
Instead of hard-coding the 0.2.2 release, would it be worth doing something like [this](https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c) to get the latest release from that repository?

Reply:
Well, I think it's better to lock the version in case of breaking changes.

One thing we could do is extract the version to a constant like so:

```Dockerfile
FROM rust:latest as build

ARG CLR_NAME=cargo-local-registry
ARG CLR_VERSION=0.2.2
ARG CLR_URL=https://github.com/dhovart/${CLR_NAME}/releases/download/${CLR_VERSION}/${CLR_NAME}-${CLR_VERSION}-x86_64-unknown-linux-musl.tar.gz

# ...

RUN curl -L -o clr.tar.gz "${CLR_URL}" && \
# ...
```

This way it's more legible and it could be changed more easily.
We can also do the same for jq.

Reply:
Updated